client/package.json

```diff
@@ -33,6 +33,8 @@
     "@react-keycloak/web": "3.4.0",
     "@tanstack/react-query": "^4.22.0",
     "@tanstack/react-query-devtools": "^4.22.0",
+    "classnames": "^2.3.1",
+    "formik": "2.2.6",
```

Comment:
I'm a bit nervous to see another form library when we already have react-hook-form.

After looking, this doesn't seem to be used.  Please remove.